### DISCUSSION TITLE
:id conflicts with other framework

### DISCUSSION THREAD

**Discussion by jaeyson** [alpinejs/alpine#3229] [1. Help] [answered]
Hi! is there a way to customize `:id` into something like `x-attr-id`? I have conflicts with using it, specifically when using [Phoenix liveview special attributes](https://hexdocs.pm/phoenix_live_view/Phoenix.Component.html#slot/3). Thanks!

**Comment by ekwoka** [comment, marked as answer]
You can do `x-bind:id`
